subprojects/packagefiles/liblzma/src/xz/meson.build

```diff
@@ -1,31 +1,33 @@
-sources = [
-  'args.c',
-  'args.h',
-  'coder.c',
-  'file_io.c',
-  'file_io.h',
-  'hardware.c',
-  'list.c',
-  'list.h',
-  'main.c',
-  'message.c',
-  'mytime.c',
-  'options.c',
-  'signals.c',
-  'suffix.c',
-  'util.c',
-  '../common/tuklib_open_stdxxx.c',
-  '../common/tuklib_progname.c',
-  '../common/tuklib_exit.c',
-  '../common/tuklib_mbstr_width.c',
-  '../common/tuklib_mbstr_fw.c',
-]
+if cc.get_id() != 'msvc'
```

Comment:
This might be simpler to do by either checking in src/meson.build before subdir'ing in, or just checking here and doing subdir_done().

IMHO it is slightly clearer to read than figuring out what is covered by the indented code, and also, it avoids having to change lots of lines just to move the indentation. :D

Reply:
Done.